Source/swiftlint/Helpers/LintableFilesVisitor.swift

```diff
@@ -28,6 +28,16 @@ enum LintOrAnalyzeModeWithCompilerArguments {
     case analyze(allCompilerInvocations: CompilerInvocations)
 }
 
+private func resolveParamsFiles(args: [String]) -> [String] {
+    return args.reduce(into: []) { allArgs, arg in
+        if arg.hasPrefix("@"), let contents = try? String(contentsOfFile: String(arg.dropFirst())) {
+            allArgs += resolveParamsFiles(args: contents.split(separator: "\n").map(String.init))
```

Comment:
Probably up to us whether or not this should actually handle these recursively. I believe compilers do.